canonical-web-and-design/vanilla-framework
# Add JS notifications and functionality to docs

## Done

Added descriptions of JS functionality, added some warnings about JS being needed.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check the code and website run ok.
